src/main/java/uk/gov/hmcts/ccd/CachingConfiguration.java

```diff
@@ -35,6 +38,7 @@ private void configCaches(int definitionCacheTTL, Config config) {
         config.addMapConfig(newMapConfig("caseTabCollectionCache", definitionCacheTTL));
         config.addMapConfig(newMapConfig("wizardPageCollectionCache", definitionCacheTTL));
         config.addMapConfig(newMapConfig("userRolesCache", definitionCacheTTL));
+        config.addMapConfig(newMapConfig("userCache", applicationParams.getDailyCacheTTLSecs()));
```

Comment:
can you add a test to the DefinitionsCachingIT please

Reply:
done